Tests/WebTest/SitemapTest.php

```diff
@@ -87,14 +90,19 @@ public function getFormats()
                          <xhtml:link href="http://localhost/sitemap-aware?_locale=de" hreflang="de" rel="alternate"/>
                      </url>
                      <url>
+                         <loc>http://localhost/sitemap-aware-last-mod-date</loc>
+                         <lastmod>2016-07-06T00:00:00+02:00</lastmod>
```

Comment:
i think you should not try to compare rendered dates, but the datetime objects themselves. not sure if a datetime object stores the timezone and if `==` looks at the timezone. otherwise get the timestamp from the two datetime objects.